packages/stores/src/tx/utils.ts

```diff
@@ -1,10 +1,5 @@
 export function isSlippageError(tx: any): boolean {
-  if (
-    tx &&
-    tx.code === 7 &&
-    tx.codespace === "gamm" &&
```

Comment:
I removed these because slippage errors can come from modules other than gamm and have a different code.

Given current chain setup, I think basic string matching is the most appropriate here